contracts/feeds/PriceFeeds.sol

```diff
@@ -11,13 +11,13 @@ import "@openzeppelin-2.5.0/token/ERC20/IERC20.sol";
 import "../interfaces/IERC20Detailed.sol";
 import "../core/Constants.sol";
 import "./IPriceFeedsExt.sol";
+import "../governance/PausableGuardian.sol";
+import "../../interfaces/IToken.sol";
+import "../utils/SignedSafeMath.sol";
 
-
-contract PriceFeeds is Constants, Ownable {
+contract PriceFeeds is Constants, Ownable, PausableGuardian {
```

Comment:
PausableGuardian already inherits Ownable, no need to inherit it for PriceFeeds.

Reply:
agree was there historically, just need to make sure storage allignment

Reply:
fixed